src/site/artifacts/mod.rs

```diff
@@ -4,7 +4,16 @@ pub mod page;
 
 use crate::config::Config;
 use crate::errors::*;
+use axohtml::dom::UnsafeTextNode;
 use axohtml::elements::div;
+use axohtml::unsafe_text;
+
+pub fn get_copy_logo() -> Box<UnsafeTextNode<String>> {
```

Comment:
can we make this get_copylogo- at first i was having a real hard time parsing

Reply:
my bad, its actually an icon so changed to get_copyicon